brainscore_language/__init__.py

```diff
@@ -69,4 +107,18 @@ def score(model_identifier: str, benchmark_identifier: str) -> Score:
     score: Score = benchmark(model)
     score.attrs['model_identifier'] = model_identifier
     score.attrs['benchmark_identifier'] = benchmark_identifier
+    save_score(score)
```

Comment:
This method should imo not save the score, I think this should go into the calling function that requires it for communication

Reply:
When running in an environment, `score` is called from bash (`python brainscore_language score --model_identifier=$MODEL_ID --benchmark_identifier=$BENCHMARK_ID`), so putting it into the calling function is not really an option.